beacon-chain/core/blocks/block_operations.go

```diff
@@ -440,6 +440,54 @@ func ProcessBlockAttestations(
 	return beaconState, nil
 }
 
+func verifyAttestationSig(beaconState *pb.BeaconState, att *pb.Attestation) error {
+	attesterIndices, err := helpers.AttestationParticipants(beaconState, att.Data, att.AggregationBitfield)
+	if err != nil {
+		return fmt.Errorf("could not get attestation participants: %v", err)
+	}
+
+	attestorPubKeys := make([]*bls.PublicKey, len(attesterIndices))
+	for idx, participantIndex := range attesterIndices {
+		pubkey, err := bls.PublicKeyFromBytes(beaconState.ValidatorRegistry[participantIndex].Pubkey)
+		if err != nil {
+			return fmt.Errorf("could not deserialize attestor public key: %v", err)
```

Comment:
you could specify what is the attestor index in the error statement
```go
fmt.Errorf("could not deserialize attestor with index %d public key: %v", participantIndex, err)
```

Reply:
Good idea!